plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -5057,19 +5057,45 @@ public void testSplitPruningForFilterOnNonPartitionColumn()
     public void testGetIcebergTableProperties()
     {
         assertUpdate("CREATE TABLE test_iceberg_get_table_props (x BIGINT)");
-        verifyIcebergTableProperties(computeActual("SELECT * FROM \"test_iceberg_get_table_props$properties\""));
+        verifyIcebergTableProperties(computeActual("SELECT * FROM \"test_iceberg_get_table_props$properties\""), HiveCompressionCodec.ZSTD);
         assertUpdate("DROP TABLE test_iceberg_get_table_props");
     }
 
-    protected void verifyIcebergTableProperties(MaterializedResult actual)
+    protected void verifyIcebergTableProperties(MaterializedResult actual, HiveCompressionCodec codec)
     {
         assertThat(actual).isNotNull();
-        MaterializedResult expected = resultBuilder(getSession())
+        Optional<String> codecName = switch (format) {
+            case AVRO -> switch (codec) {
+                case NONE -> Optional.of("uncompressed");
+                case SNAPPY -> Optional.of("snappy");
+                case LZ4 -> Optional.empty(); // codec unsupported by Iceberg
```

Comment:
throw new IllegalStateException("Unsupported LZ4 code for AVRO file format");

Reply:
If we throw here, we won't be able to assert table creation fails